src/test_utils.jl

```diff
@@ -168,7 +168,7 @@ function test_interface(
     inputs=rand(5, 5), outputs=rand(3, 5)
 )
     predictions = test_common(template, inputs, outputs)
-    @test predictions isa Vector{<:MultivariateNormal{<:Real}}
```

Comment:
Why do these have to be `Continuous`?

Reply:
Because multivariate classification isn't a thing we care about supporting?
We barely support classification.